### DISCUSSION TITLE
ErrorException

### DISCUSSION THREAD

**Discussion by netsolid-ca** [Power-Components/livewire-powergrid#1478] [Q&A] [answered]
I keep running into an error after trying to integrate Powergrid into my laravel application.

Attempt to read property "layout" on array

Its referring to: 
>> resources / views / vendor / livewire-powergrid / components / frameworks / tailwind / table-base.blade.php: 14

>> @include($theme->layout->header, [
>>                'enabledFilters' => $enabledFilters,
>>            ])

Despite my numerous attempts to resolve the problem including
>>php artisan vendor:publish --tag=livewire-powergrid-views
>>php artisan optimze:clear
>>npm run build or npm run dev --host
>>verifying all config files

I cannot get to the bottom of this, please let me know if there is anything I could be missing.

**Comment by marineusde** [comment, marked as answer]
do you publish the blade files? What powergrid version do you use?

the code in the current version of 5.x is:

```
@include(data_get($theme, 'layout.header'), [
  'enabledFilters' => $enabledFilters,
])
```
